Doc/library/tkinter.rst

```diff
@@ -464,12 +464,11 @@ The Packer
 .. index:: single: packing (widgets)
 
 The packer is one of Tk's geometry-management mechanisms.    Geometry managers
-are used to specify the relative positioning of the positioning of widgets
-within their container - their mutual *master*.  In contrast to the more
-cumbersome *placer* (which is used less commonly, and we do not cover here), the
-packer takes qualitative relationship specification - *above*, *to the left of*,
-*filling*, etc - and works everything out to determine the exact placement
-coordinates for you.
+are used to specify the relative positioning of widgets within their container -
```

Comment:
For the future: the source .rst files are not required to have their lines nicely filled out before being wrapped.  You could have merely removed the extra from one line and left the rest alone.   I then would have been required to check only the one line.

Reply:
Thank you for pointing that out @terryjreedy. I appreciate your guidance and will remember that for next time!